test/unit/spec/MfaVerify_spec.js

```diff
@@ -174,6 +174,39 @@ function (Okta,
       });
     }
 
+    function setupWebauthnOnly() {
+      var setNextResponse = Util.mockAjax();
+      var baseUrl = 'https://foo.com';
+      var authClient = new OktaAuth({url: baseUrl, transformErrorXHR: LoginUtil.transformErrorXHR});
+      var successSpy = jasmine.createSpy('success');
+      var router = new Router(_.extend({
```

Comment:
you can change, at the top of the file, the allowed max-statements, and set it to 35, or whatever number you need